Improve DateBox demo - Overview (#527)
DevExpress/devextreme-demos: JSDemos/Demos/DateBox/Overview/description.md

> The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) is an editor that allows users to select date and time and displays them. This demo illustrates the following **DateBox** properties:

Reviewer: If possible, I would always try to avoid definitions, i.e. "This is control is a control that allows you to do smth...". A lot is obvious based on the control name itself. But that's not as important as the sentence structure... The X is Y that allows you to Z - this sounds too formal (people already criticize our language for being too "boring, academic"). This type of sentence could probably work if you need to introduce a complicated term - but we actually try to make it all sound nice and easy. 

```suggestion
The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) editor helps users enter or modify date and time values. This demo illustrates the following **DateBox** properties:
```

> The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) displays date and time in DeveExtreme [predefined formats](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/#type) or a formats provided by an external localization library such as [Intl](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies/Using_Intl). This demo illustrates the following  **DateBox** properties:

Reviewer 2: It would be better to mention different formats in the displayFormat description instead.

```suggestion
The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) is an editor that allows users to select date and time and displays them. This demo illustrates the following **DateBox** properties:
```

- [value](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#value)     
A value the **DateBox** displays.

- [type](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#type)       

> Specifies whether the **DateBox** allows users to select only date, only time, or both date and time.

Reviewer: ```suggestion
Specifies whether the **DateBox** allows users to select only date, only time, or both.
```

> Specifies whether the **DateBox** displays the date, time, or both date and time.

Reviewer 2: ```suggestion
Specifies whether the **DateBox** allows users to select only date, only time, or both date and time.
```

- [displayFormat](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat)        

> **DateBox** date display format.

Reviewer 2: ```suggestion
A date/time display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter case.
```

> A date/time display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter case.

Reviewer: ```suggestion
A date/time display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter.
```

- [pickerType](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#pickerType)        

> The type of date/time picker.

Reviewer: There's already a "type" property described above. Now here's a picker type. In the other item you explained what the "type" was. Here you don't mention what the word "type" means. So maybe we should add some details:

```suggestion
Specifies the type of UI used to select date or time. This demo shows how to change calendar to roller date picker. 
```

> Specifies the type of UI used to select date or time. This demo shows how to change calendar to roller date picker. 

Reviewer 3: What do you mean by "roller date picker"? I haven't seen "roller" used in this context.

Author: "Rollers" is the term used in our documentation https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#pickerType
It looks like a native date picker and allows users to scroll through dates.

Reviewer 3: Hi Anton, I responded to you in our MS Teams chat.

- [showClearButton](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#showClearButton)        
Specifies whether to display the button that clears the **DateBox** **value**.

- [disabled](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#disabled)        
Specifies whether the DateBox responds to user interaction.

- [disabledDates](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#disabledDates)      
Dates that are not available for selection.

- [applyValueMode](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#applyValueMode)     

> Defines how users apply the selected **DateBox** value.

Reviewer 2: ```suggestion
Defines whether the selected value applies instantly or after a user clicks an Apply button.
```

> Defines whether the selected value applies instantly or after a user clicks an Apply button.

Reviewer: ```suggestion
Defines whether the selected value applies instantly or after a user clicks Apply.
```
or 

```suggestion
Defines whether the selected value applies instantly or after a user clicks the Apply button.
```

> Defines whether the selected value applies instantly or after a user clicks the Apply button.

Reviewer 3: ```suggestion
Defines whether the selected value applies instantly or after a user clicks the **Apply** button.
```
